readme.md

```diff
@@ -1,156 +1,102 @@
-![Renovate banner](https://app.renovatebot.com/images/renovate_660_220.jpg)
+![Renovate banner](images/cli_banner.png)
 
-# Renovate
+# What is Renovate?
 
-Automated dependency updates.
-Multi-platform and multi-language.
+Renovate is a dependency update tool. It helps to update dependencies in your code to the latest version.
+When Renovate runs on your repo, it looks for all known references to external dependencies and, if there are newer versions available, Renovate will create a pull request to update your code directly.
 
-[![License: AGPL-3.0-only](https://img.shields.io/badge/license-%20%09AGPL--3.0--only-blue.svg)](https://raw.githubusercontent.com/renovatebot/renovate/main/license)
-[![codecov](https://codecov.io/gh/renovatebot/renovate/branch/main/graph/badge.svg)](https://codecov.io/gh/renovatebot/renovate)
-[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
-[![Build status](https://github.com/renovatebot/renovate/actions/workflows/build.yml/badge.svg)](https://github.com/renovatebot/renovate/actions/workflows/build.yml)
-![Docker Pulls](https://img.shields.io/docker/pulls/renovate/renovate?color=turquoise)
-[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/renovatebot/renovate/badge)](https://securityscorecards.dev/viewer/?uri=github.com/renovatebot/renovate)
+**Features**
+- Delivers updates directly to your code base
```

Comment:
This may confuse people. Updates generally are done via PR and require review by developers. You may worry them that it's fully automated and they don't get to approve

Reply:
True. I just meant - your code / your repo / your source code. "directly to the repo" is probably a better way to say it. A PR is still "directly to the repo".

Reply:
Updated to "delivers update PRs directly to your repo"